py/torch_tensorrt/ts/_compiler.py

```diff
@@ -156,7 +156,7 @@ def convert_method_to_trt_engine(
     dla_global_dram_size=536870912,
     truncate_long_and_double=False,
     calibrator=None,
-) -> str:
+) -> bytearray:
```

Comment:
Could change the `method_name` argument in this function to default to `forward`